packages/webview_flutter/ios/Classes/FLTWKProgressionDelegate.m

```diff
@@ -0,0 +1,45 @@
+//
+//  FLTWKProgressionDelegate.m
+//  webview_flutter
+//
+//  Created by Jérémie Vincke on 03/10/2019.
+//
+
+#import "FLTWKProgressionDelegate.h"
+
+NSString *const keyPath = @"estimatedProgress";
+
+@implementation FLTWKProgressionDelegate {
+  FlutterMethodChannel *_methodChannel;
+}
+
+- (instancetype)initWithWebView:(WKWebView *)webView channel:(FlutterMethodChannel *)channel {
+  self = [super init];
+  if (self) {
+    _methodChannel = channel;
+    [webView addObserver:self
+              forKeyPath:keyPath
+                 options:NSKeyValueObservingOptionNew | NSKeyValueObservingOptionOld
```

Comment:
Do we need to observe the old key here?

Reply:
You're right, observing the old value isn't useful here. I'll update the changes